packages/office-addin-dev-settings/src/dev-settings-windows.ts

```diff
@@ -16,6 +16,11 @@ const SourceBundlePort: string = "SourceBundlePort";
 const UseDirectDebugger: string = "UseDirectDebugger";
 const UseLiveReload: string = "UseLiveReload";
 const UseProxyDebugger: string = "UseWebDebugger";
+const WebViewSelection: string = "WebViewSelection";
+const IE: string = "ie";
```

Comment:
It would help to use a common prefix for these strings, e.g. `const WebViewTypeString_IE = "ie";  const WebViewTypeString_Edge = "edge"; etc.